docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,380 @@
+
+- [Data Grid](#data-grid)
+  - [Pivoting](#pivoting)[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+  - [AI Assistant – Featuring LLMs on the Data Grid](#ai-assistant-featuring-llms-on-the-data-grid)[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+  - [List View](#list-view)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [Server-Side Aggregation and Lazy Loading](#server-side-aggregation-and-lazy-loading)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [Data Source on the Community Plan](#data-source-on-the-community-plan)
+  - [New Toolbar](#new-toolbar)
+  - [Design Agnostic Core](#design-agnostic-core)
+  - [New No Columns Overlay](#new-no-columns-overlay)
+  - [Performance Improvements](#performance-improvements)
+- [Date and Time Pickers](#date-and-time-pickers)
+  - [Time Range Picker](#time-range-picker)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [Auto Switch Strategy](#auto-switch-strategy)
+  - [Clear `ownerState` for slots](#clear-ownerstate-for-slots)
+  - [Accessible DOM Structure](#accessible-dom-structure)
+  - [Keyboard Editing on Mobile Pickers](#keyboard-editing-on-mobile-pickers)
+- [Tree View](#tree-view)
+  - [Lazy loading child nodes](#lazy-loading-child-nodes) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [Automatic selection propagation](#automatic-selection-propagation)
+  - [New Customization Hook](#new-customization-hook)
+- [Charts](#charts)
+  - [Funnel Charts](#funnel-charts)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [Radar Charts](#radar-charts)
+  - [Server-Side Rendering for Charts](#server-side-rendering-for-charts)
+  - [Refined Design and Interaction](#refined-design-and-interaction)
+  - [HTML Legend for Charts](#html-legend-for-charts)
+  - [Charts Composition](#charts-composition)
+- [Enhanced License Management and Telemetry](#enhanced-license-management-and-telemetry)
+- [Priority Support – A New Support Service](#priority-support-a-new-support-service)
+- [Support for Material UI](#support-for-material-ui)
+- [Migration](#migration)
+- [Long-Term Support](#long-term-support)
+- [What's next](#whats-next)
+- [How to Get Involved](#how-to-get-involved)
+
+## Data Grid
+
+### Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+The wait is over! With pivoting, your users can easily restructure and summarize large datasets: group related data, calculate aggregates (such as sums, averages, and counts), and compare different data categories side by side. This flexibility enables faster insight discovery and more efficient data analysis.
+
+The feature is available out-of-the-box, but you can also create a pre-configured setup depending on your user requirements.
```

Comment:
You would only hyphenate "out of the box" if you were using it as an adjective, such as "out-of-the-box features."

```suggestion
This feature is available right out of the box, but you can also create a preconfigured setup to tailor it to your user requirements.
```